image/cli/mascli/templates/pipelinerun-update.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       value: '$MONGODB_NAMESPACE'
     - name: mongodb_v5_upgrade
       value: '$MONGODB_V5_UPGRADE'
+    - name: mongodb_v6_upgrade
+      value: '$MONGODB_V6_UPGRADE'
```

Comment:
Do we need to update the documentation and help for this new   `mongodb_v6_upgrade` and `mongodb_replicas`?

Reply:
Thanks added a fix for it.